[2025-04-05T09:50:43.574+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_and_insert_data_to_db.create_table_task manual__2025-04-05T09:50:42.815826+00:00 [queued]>
[2025-04-05T09:50:43.578+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_and_insert_data_to_db.create_table_task manual__2025-04-05T09:50:42.815826+00:00 [queued]>
[2025-04-05T09:50:43.578+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-04-05T09:50:43.642+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): create_table_task> on 2025-04-05 09:50:42.815826+00:00
[2025-04-05T09:50:43.645+0000] {standard_task_runner.py:57} INFO - Started process 489 to run task
[2025-04-05T09:50:43.649+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'read_and_insert_data_to_db', 'create_table_task', 'manual__2025-04-05T09:50:42.815826+00:00', '--job-id', '1262', '--raw', '--subdir', 'DAGS_FOLDER/news_dags.py', '--cfg-path', '/tmp/tmpyhpuk4gd']
[2025-04-05T09:50:43.652+0000] {standard_task_runner.py:85} INFO - Job 1262: Subtask create_table_task
[2025-04-05T09:50:43.679+0000] {task_command.py:415} INFO - Running <TaskInstance: read_and_insert_data_to_db.create_table_task manual__2025-04-05T09:50:42.815826+00:00 [running]> on host 39101a2f689c
[2025-04-05T09:50:43.723+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='read_and_insert_data_to_db' AIRFLOW_CTX_TASK_ID='create_table_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T09:50:42.815826+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T09:50:42.815826+00:00'
[2025-04-05T09:50:43.733+0000] {logging_mixin.py:151} INFO - ✅ Đã tạo bảng 'tintuc' (nếu chưa tồn tại).
[2025-04-05T09:50:43.733+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-05T09:50:43.740+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=read_and_insert_data_to_db, task_id=create_table_task, execution_date=20250405T095042, start_date=20250405T095043, end_date=20250405T095043
[2025-04-05T09:50:43.753+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-05T09:50:43.765+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
